config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-external-etcd.yaml

```diff
@@ -0,0 +1,65 @@
+# periodic jobs
+
+periodics:
+- name: ci-kubernetes-e2e-kubeadm-kinder-external-etcd-master
+  interval: 2h
+  decorate: true
+  labels:
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+  decoration_config:
+    timeout: 40m
+  extra_refs:
+  - org: kubernetes
+    repo: kubernetes
+    base_ref: master
+    path_alias: k8s.io/kubernetes
+  - org: kubernetes
+    repo: kubeadm
+    base_ref: master
+    path_alias: k8s.io/kubeadm
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20190605-45f60c3-master
+      command:
+      - runner.sh
+      - "../kubeadm/kinder/ci/kinder-run.sh"
+      args:
+      - "external-etcd-master"
+      securityContext:
+        privileged: true
+      resources:
+        requests:
+          memory: "9000Mi"
+          cpu: 2000m
+- name: ci-kubernetes-e2e-kubeadm-kinder-external-etcd-1-14
+  interval: 12h
+  decorate: true
+  labels:
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+  decoration_config:
+    timeout: 40m
+  extra_refs:
+  - org: kubernetes
+    repo: kubernetes
+    base_ref: release-1.14
+    path_alias: k8s.io/kubernetes
+  - org: kubernetes
+    repo: kubeadm
+    base_ref: master
+    path_alias: k8s.io/kubeadm
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:latest-1.14
```

Comment:
https://console.cloud.google.com/gcr/images/k8s-testimages/GLOBAL/kubekins-e2e?gcrImageListsize=30
technically this should be `v20190606-4bd9b45-1.14`
it will also solve the CI failure (if `latest` is used it needs `imagePullPolicy: Always`

we might have update the kubekins tags in the other kinder YAML files.
but this is for separate PRs.

Reply:
ok, let's have this as `v20190605-45f60c3-1.14`
our tags seem off for the latest for some reason, no idea why.

Reply:
Updated, thanks!